### PR TITLE
Factor out log message.

### PR DESCRIPTION
**Pull request by jerome-benoit**
+ Imports sorting

### CODE REVIEW

**Comment by LucasBrazi06** [comment]
> * Imports sorting



> * Imports sorting

I had issue with the sort import plugin in VS code that broke the build. I cannot reproduce today. I'll let you know.

File: src/server/rest/CentralRestServer.ts [hunk 1]
```diff
@@ -99,15 +99,15 @@ export default class CentralRestServer {
...
     Logging.logInfo({
       tenantID: Constants.DEFAULT_TENANT,
       module: MODULE_NAME, method: 'startSocketIO',
       action: ServerAction.STARTUP,
-      message: logMsg
+      message: logMsg + '...'
     });
     // eslint-disable-next-line no-console
-    console.log(`Starting REST SocketIO Server ${cluster.isWorker ? 'in worker ' + cluster.worker.id : 'in master...'}`);
+    console.log(`${logMsg} ${cluster.isWorker ? 'in worker ' + cluster.worker.id.toString() : 'in master...'}`);
```
Comment (LucasBrazi06) [inline src/server/rest/CentralRestServer.ts:110]: No need this kind of optimization, I want to read the message without looking for a var

Comment (jerome-benoit) [reply src/server/rest/CentralRestServer.ts:110]: It's 10 lines up ;)

**Comment by jerome-benoit** [comment]
> > * Imports sorting
> 
> I had issue with the sort import plugin in VS code that broke the build. I cannot reproduce today. I'll let you know.

Disable the plugin until a new version is released with that fix in: https://github.com/amatiasq/vsc-sort-imports/pull/69. A npm script is doing it properly instead.

**Comment by LucasBrazi06** [comment]
> Disable the plugin until a new version is released with that fix in: [amatiasq/vsc-sort-imports#69](https://github.com/amatiasq/vsc-sort-imports/pull/69). A npm script is doing it properly instead.

Ok thanks!

File: src/server/rest/CentralRestServer.ts (hunk 1 above)
Comment (LucasBrazi06) [reply src/server/rest/CentralRestServer.ts:110]: > It's 10 lines up ;)

Ok for that one LOL.

**Pull request merged**